### PR TITLE
Add dataset transformer output to node

### PR DESCRIPTION
**Pull request by blanchco**
# Description

* Add a single output to the datatransformer node, this can be extended to show multiple outputs in the future if needed.
* `on-selected` to response component to select the node
* `update-selected-output` to chat component to update which outputs are selected which are stored in the state.  These state outputs can then be accessed to be shown on the node

https://github.com/user-attachments/assets/62177ce7-b328-477c-b149-6481de1fc220

### CODE REVIEW

File: packages/client/hmi-client/src/components/llm/tera-beaker-code-cell-output.vue
Comment (YohannParis) [inline packages/client/hmi-client/src/components/llm/tera-beaker-code-cell-output.vue:1]: Nice CSS job here!

File: packages/client/hmi-client/src/components/workflow/ops/dataset-transformer/tera-dataset-transformer-node.vue [hunk 1]
```diff
@@ -3,7 +3,17 @@
...
-		<Button v-else label="Edit" @click="emit('open-drilldown')" severity="secondary" outlined />
+		<template v-if="node.inputs[0].value">
+			<template v-if="!isEmpty(node.state.selectedOutputs)">
+				<tera-beaker-code-cell-output
+					v-for="m in node.state.selectedOutputs?.[0]?.messages"
+					:key="m.header.msg_id"
+					:jupyter-message="m"
+					:feature-config="{ isPreview: true }"
+				/>
+			</template>
```
Comment (YohannParis) [inline packages/client/hmi-client/src/components/workflow/ops/dataset-transformer/tera-dataset-transformer-node.vue:14]:
```suggestion
				<tera-beaker-code-cell-output
				  v-if="!isEmpty(node.state.selectedOutputs)"
					v-for="m in node.state.selectedOutputs?.[0]?.messages"
					:key="m.header.msg_id"
					:jupyter-message="m"
					:feature-config="{ isPreview: true }"
				/>
```

**Review by YohannParis — APPROVED** [review]
LGTM

File: packages/client/hmi-client/src/components/workflow/ops/dataset-transformer/tera-dataset-transformer-node.vue (hunk 1 above)
Comment (blanchco) [reply packages/client/hmi-client/src/components/workflow/ops/dataset-transformer/tera-dataset-transformer-node.vue:14]: I think using `v-if` and `v-for` on the same component is not recommended by Vue which is why I had the `template` tag there

Comment (YohannParis) [reply packages/client/hmi-client/src/components/workflow/ops/dataset-transformer/tera-dataset-transformer-node.vue:14]: Why not do `v-for="m in (node.state.selectedOutputs?.[0]?.messages ?? [])" then and avoid the v-if?

Comment (blanchco) [reply packages/client/hmi-client/src/components/workflow/ops/dataset-transformer/tera-dataset-transformer-node.vue]: that works, I changed this up in my latest commit

**Pull request merged**